docs/split-inputs.md

```diff
@@ -119,9 +119,70 @@ It is relatively easy for a modern fuzzing engine to discover the separator,
 but nevertheless we recommend to provide several seed inputs with the desired number
 of separators.
 
-## Fuzz Data Provider
-
-TODO
+## Fuzzed Data Provider
+
+[FuzzedDataProvider] is a single-header C++ library that is helpful for
+splitting a fuzz input into multiple parts of various types. Both advantage and
+disadvantage of using this library is that the input splitting happens
+dynamically, i.e. you don't need to define any structure of the input. This
+might be very helpful in certain cases, but could also make the corpus human
```

Comment:
Perhaps also mention that the corpus might no longer be in a particular file format? E.g., for image/document parsers, the corpus files won't be image / document files anymore.

Reply:
Thanks, Stefan, that's a much better wording than "human unreadable". Fixed!